contracts/actions/curve/CurveDeposit.sol

```diff
@@ -0,0 +1,145 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity =0.7.6;
+pragma experimental ABIEncoderV2;
+
+import "./helpers/CurveHelper.sol";
+import "../../utils/TokenUtils.sol";
+import "../../utils/SafeMath.sol";
+import "../ActionBase.sol";
+
+contract CurveDeposit is ActionBase, CurveHelper {
+    using TokenUtils for address;
+    using SafeMath for uint256;
+
+    function _sig(uint256 _nCoins, bool _useUnderlying) internal pure returns(bytes4) {
+        if (!_useUnderlying) {
+            if (_nCoins == 2) return bytes4(0x0b4c7e4d);
```

Comment:
keccak(string) to see the funcSig in readable form

Reply:
done, moved to CurveView